src/app/onion/learning-object-builder/components/outcome/outcome-typeahead/outcome-typeahead.component.ts

```diff
@@ -100,9 +95,11 @@ export class OutcomeTypeaheadComponent implements OnInit, OnChanges, OnDestroy {
           this.goodVerb = false;
         }
 
-        if (this.goodVerb) {
-          this.enteredText.emit(this.text);
-        }
+        // we want to always emit changes from the outcome fields even if they are null
+        // there's a function that will validate each field and will throw an error if needed
+        this.enteredText.emit(this.text);
+        this.selectedVerb.emit(this.verb);
+        this.selectedCategory.emit(this.bloom);
```

Comment:
Does this work when you input an incorrect verb such as 'asdf'?

Reply:
Yes, we emit the verb, but the builder-store handles the validation logic